firebase.js

```diff
@@ -0,0 +1,10 @@
+// imports
+const admin = require('firebase-admin');
+
+// init firebase-admin
+require('dotenv').config();
+const credentials = require('./firebase-adminsdk.json');
```

Comment:
🚫 **[eslint]** <[import/newline-after-import](https://github.com/import-js/eslint-plugin-import/blob/v2.26.0/docs/rules/newline-after-import.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Expected 1 empty line after require statement not followed by another require.
```suggestion
const credentials = require('./firebase-adminsdk.json');

```